helpers/filterSeats.js

```diff
@@ -60,19 +66,58 @@ module.exports = function(dust) {
       });
     }
 
+    // get the names of the best two sections from the theatre
+    function getBestSections(bandColours) {
+      var bestSections = [];
+      // add all section names to an array
+      _.forEach(bandColours, function(bands, section) {
+        bestSections.push(section);
+      });
+      // find the best section
+      bestSections = _.last(bestSections); // just Stalls
+      return bestSections;
+    }
+
+    // compare prices
+    function chosenForYou(ticketRate, ticketRateSection) {
+      var bestSections = getBestSections(params.bandColours);
+      var ticketRatePrice = ticketRates[ticketRate.ids].grossPrice;
+      // check if current ticketRate exists in
+      if (bestSections === ticketRateSection && _.last(ticketRate.colourRank)) {
+        // check if price is cheaper or exists
```

Comment:
Being picky here, but the comment is the wrong way around I think?
You're checking first if the price exists, after that if the price is lower than the other.